app/views/case_contacts/_case_contact.html.erb

```diff
@@ -7,11 +7,10 @@
           <div class="media-body">
             <h5 class="mt-0 card-title">
               <strong class="text-primary">
-              <%= link_to(t("button.undelete"), restore_case_contact_path(contact.id), method: :post, class: "btn btn-info") if contact.deleted? %>
```

Comment:
@compwron I don't know why this line is here 😅 I double-checked my code and there's only the undelete button below. If anything changed lately, please let me know. Now I just delete it because it look weird :D. Thank you

Reply:
that's weird
I think you're doing the right thing :)